vnext/Microsoft.ReactNative/ReactHost/ReactInstanceWin.cpp

```diff
@@ -242,9 +246,40 @@ ReactInstanceWin::ReactInstanceWin(
   // OnInstanceCreated event is raised only after the internal react-native instance is ready and
   // it starts handling JS queue work items.
   m_whenCreated.SetValue();
+
+  if (options.EnableDefaultCrashHandler()) {
+    CrashManager::SetupCustomHander();
```

Comment:
This looks like a global state, rather than anything instance specific.  -- Why is this tried to a ReactInstance Settings, rather than just being a static function that should be called at a per app level?

Reply:
Do we have existing examples of that? Is it common for apps to have more than one ReactNativeHost / ReactInstance?

Reply:
Its not common, but Office for one.

Reply:
Figured out a cleaner design where we inc/dec for each react instance that opts in. That will cover cases where apps have multiple instances with differing options.